whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/BaseEndToEndSpec.groovy

```diff
@@ -237,11 +237,18 @@ ${notification.contents}
     }
 
     String syncUpdate(String content) {
-        syncUpdate(content, null, false)
+        syncUpdate(content, null, false, null)
     }
 
-    String syncUpdate(String content, String charset, boolean notifications) {
-        def response = syncUpdate(new SyncUpdate(data: content, charset: charset))
+    String syncUpdateHttp(String content) {
```

Comment:
Do you need this specific method if it's only called once?

Reply:
I can call directly syncupdates